src/utils/account.js

```diff
@@ -38,15 +35,11 @@ export const isAddress = address => crypto.checkAddress(address, 'panacea');
 
 export const getAccountFromPrivKey = privateKey => new Account({ privateKey });
 
-export const getAccountFromKeyPair = (keyPair, password) => {
-  const encryptedPrivKey = cryptography.encryptKey(password, keyPair.privKey);
-  return new local.Account(password, encryptedPrivKey, keyPair.pubKey);
+export const getAccountFromEncKey = (encKey, password) => {
+  const privateKey = crypto.getPrivateKeyFromKeyStore(encKey, password);
+  return getAccountFromPrivKey(privateKey);
```

Comment:
We don't need to use the `cryptography` from `medjs` which is [the old-old Panacea JS SDK](https://github.com/medibloc/panacea-js-old). 

I checked that the `encKey` parameter is in Etheruem Web3 Secret Storage which can be accepted by `crypto.getPrivateKeyFromKeyStore()` of the new `panacea-js`.

Reply:
I also found that `medjs` is used in another file, and I think that file is not necessary anymore. 
I mean that features in that file aren't exposed to the web UI (such as, vesting, voting, ...).
I've also tried to remove those unnecessary craps, but it was so hard to navigate codes.

I guess it's much easier to write everything from scratch, instead of removing codes.